pkg/web_app/lib/src/account.dart

```diff
@@ -182,20 +179,22 @@ class _PkgAdminWidget {
       _toggleDiscontinuedButton.onClick.listen((_) => _toogleDiscontinued());
       _setPublisherButton?.onClick?.listen((_) => _setPublisher());
     }
-    update();
   }
 
   Future _toogleDiscontinued() async {
-    final confirmed = await modalConfirm(
-        'Are you sure you want change the "discontinued" status of the package?');
-    if (!confirmed) return;
-
-    final options =
-        PkgOptions(isDiscontinued: !pageData.pkgData.isDiscontinued);
-    await _rpc(() async {
-      await client.setPackageOptions(pageData.pkgData.package, options);
-      window.location.reload();
-    });
+    await rpc(
+      confirmQuestion:
+          'Are you sure you want change the "discontinued" status of the package?',
+      fn: () async {
+        final options =
+            PkgOptions(isDiscontinued: !pageData.pkgData.isDiscontinued);
+        await client.setPackageOptions(pageData.pkgData.package, options);
```

Comment:
nit,
```
        await client.setPackageOptions(pageData.pkgData.package, PkgOptions(
          isDiscontinued: !pageData.pkgData.isDiscontinued,
        ));
```

Maybe it's just me... but this is a case where I love trailing comma, and avoid introducing variable I don't need :)